### DISCUSSION TITLE
Reset LSPosed / delete old files after uninstalling

### DISCUSSION THREAD

**Discussion by Inervo** [LSPosed/LSPosed#2278] [Q&A] [answered]
Hi,

My LSPosed worked perfectly until this morning, and i switch my sim card.
Since then, my LSPosed shortcut disappear. No big deal, i went to LSPosed to reinstall it.
But the option is grayed out...

![Screenshot_2022-12-05-15-24-18-10_084ab3bc34f7aa7ccff83653f52abafc](https://user-images.githubusercontent.com/2921547/205663622-8d09ccc0-1e1c-4e9c-ae1a-29c09e06bd33.jpg)

Changing back my sim card didn't change anything, i've tried to desinstall, reboot and reinstall LSPosed. i've also tried to delete /data/adb/lspd. It seems there's a persistent setting stored somewhere, because this option remain grayed out...

So my question: how can i fully reset, not the modules, but LSPosed itself to factory settings?
I want to have access to this setting, and I don't see another option..

**Comment by Inervo** [comment, marked as answer]
Found the answer by myself. I had to clear the cache and restart the launcher, and then the shortcut reappeared.

**Comment by furopi** [comment]
So after all it stopped working 😞
I finally managed to get the manager up and running again. It was hours of work... The solution I found was to use the app "Unnotification" to make the notification visible again and then restore the shortcut from the settings of the manager. Unnotification was once in the Play Store, but sadly isn't anymore... But the APK can be found on sites like apkpure! Hope this helps at least somebody! 

Cheers! 🫡
Florian
